Desktop_Interface/isodriver.cpp

```diff
@@ -255,125 +248,113 @@ void isoDriver::setVisible_CH2(bool visible){
     axes->graph(1)->setVisible(visible);
 }
 
-void isoDriver::setVoltageRange(QWheelEvent *event){
+void isoDriver::setVoltageRange(QWheelEvent* event)
+{
     if(doNotTouchGraph && !fileModeEnabled) return;
 
+    bool isProperlyPaused = properlyPaused();
+    double maxWindowSize = fileModeEnabled ? daq_maxWindowSize : ((double)MAX_WINDOW_SIZE);
+
+    display.setVoltageRange(event, isProperlyPaused, maxWindowSize, axes);
+
+    if (!(event->modifiers() == Qt::ControlModifier))
+        if (autoGainEnabled && !isProperlyPaused)
+            autoGain();
+}
+
+void DisplayControl::setVoltageRange (QWheelEvent* event, bool isProperlyPaused, double maxWindowSize, QCustomPlot* axes)
+{
     if (!(event->modifiers() == Qt::ControlModifier)){
         double c = (topRange - botRange) / (double)400;
 
         QCPRange range = axes->yAxis->range();
 
-        double pixPct = (double)100 - ((double)100 * (((double)axes->yAxis->pixelToCoord(event->y())-range.lower) / (double)(range.upper - range.lower)));
-        if (pixPct<0) pixPct = 0;
-        if (pixPct>100) pixPct = 100;
-
+        double pixPct = (double)100 - ((double)100 * (((double)axes->yAxis->pixelToCoord(event->y())-range.lower) / range.size()));
+        if (pixPct < 0) pixPct = 0; 
```

Comment:
Would prefer separate lines for if() statements, if possible.  Please don't repeat my mistakes!

Reply:
Actually, I think formatting stuff should go into a single giant review.  :)  See the `clang-format` review.